packages/styles/package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "@canonical/styles",
+  "description": "Standard utility functions for Canonical's Web Engineering team",
```

Looks like the pacage description needs to be updated, it was copied from the utils package
```suggestion
  "description": "Canonical's base styles",
```